src/simularium/localSimulators/PointSimulator.ts

```diff
@@ -128,6 +128,24 @@ export default class PointSim implements IClientSimulatorImpl {
                 y: 12,
                 z: 12,
             },
+            cameraDefault: {
+                position: {
+                    x: 0,
+                    y: 0,
+                    z: 120,
+                },
+                lookAtPosition: {
+                    x: 0,
+                    y: 0,
+                    z: 0,
+                },
+                upVector: {
+                    x: 0,
+                    y: 1,
+                    z: 0,
+                },
+                fovDegrees: 75,
+            },
```

Comment:
this z=120, fov=75 default camera could go in a constants file as DEFAULT_CAMERA_SPEC